### PR TITLE
Add method for sorting Observations into `code` and `value`

### LINKED ISSUE #1238: Add PHDC builder method for sorting Observations into code, value, and translation

### PR DESCRIPTION
**Pull request by m-goggins**
# PULL REQUEST

## Summary
Adds a method `_sort_observation` that will minimally sort values parsed from observations in a FHIR bundle into the appropriate `CodedElement` within an `Observation` during the construction of the PHDC.

## Related Issue
Fixes #1238 

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/message-parser/app/default_schemas/phdc_case_report_schema.json
Comment (robertmitchellv) [inline containers/message-parser/app/default_schemas/phdc_case_report_schema.json:1]: these renames are much cleaner imo 🥇

File: containers/message-parser/app/phdc/builder.py [hunk 1]
```diff
@@ -368,6 +371,40 @@ def _build_observation(self, observation: Observation) -> ET.Element:
 
         return entry_data
 
+    def _sort_observation(self, observation: Observation) -> Observation:
```
Comment (robertmitchellv) [inline containers/message-parser/app/phdc/builder.py:374]: this is great 🚀

**Review by robertmitchellv — APPROVED** [review]
this looks great 🎉

File: containers/message-parser/app/default_schemas/phdc_case_report_schema.json
Comment (m-goggins) [reply containers/message-parser/app/default_schemas/phdc_case_report_schema.json:1]: That's good to hear! I thought they looked wonky but do make it a little clear where the values will end up in the PHDC.

**Pull request merged**